examples/nequip/run_nequip.ipynb

```diff
@@ -0,0 +1,196 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example: using OpenMM to run a simulation with a NequIP ML potential\n",
+    "\n",
+    "You can run this example directly in your browser: [![Open On Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/sef43/openmm-ml/blob/nequip/examples/nequip/run_nequip.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Conda\n",
+    "\n",
+    "[Conda](https://docs.conda.io/) is a package and environment manager. On Google Colab, Conda is installed with [conda-colab](https://github.com/jaimergp/condacolab). On your computer, you should follow these [installation instructions](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).\n",
+    "\n",
+    "⚠️ Do not use conda-colab on your computer!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q condacolab\n",
+    "import condacolab\n",
+    "\n",
+    "condacolab.install_mambaforge()  # use mamba on colab because it is faster than conda"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install software\n",
+    "\n",
+    "First install everything we can from [conda-forge](https://conda-forge.org/).\n",
+    "Then use pip.\n",
+    "\n",
+    "⚠️ The installation might take up to 10 min!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# https://github.com/openmm/openmm-torch/issues/88\n",
+    "%env CONDA_OVERRIDE_CUDA=12.0\n",
+    "!mamba install -c conda-forge openmm-torch nnpops pytorch=*=cuda*\n",
+    "\n",
+    "!pip install git+https://github.com/mir-group/nequip@develop\n",
+    "!pip install git+https://github.com/sef43/openmm-ml@nequip"
```

Comment:
Let's not have the example install OpenMM-ML from your personal fork!  Remember that whatever you do in the example, users will copy it.

Reply:
Fixed. And thanks for the heads up, will keep that in mind :)